src/kate_with_ai/kate_with_ai_config_page.cc

```diff
@@ -19,13 +19,13 @@ kate_with_ai_config_page::kate_with_ai_config_page(QWidget * parent) : KTextEdit
   );
   }
 
-void kate_with_ai_config_page::emit_changed() { emit changed(); }
+void kate_with_ai_config_page::emit_changed() { Q_EMIT changed(); }
```

Comment:
same

Reply:
See https://doc.qt.io/qt-6/signalsandslots.html
from examples for qt6 ...
```
#include <QObject>

class Counter : public QObject
{
    Q_OBJECT

public slots:
    void setValue(int value);

signals:
    void valueChanged(int newValue);
};
```
and
```
void Counter::setValue(int value)
{
    if (value != m_value) {
        m_value = value;
        emit valueChanged(value);
    }
}
```

Reply:
I agree with all of above but it doesn't work/compile.  
It seems to be common KF6 fashion to use these not-that-obsolete-yet [macros](https://invent.kde.org/kdevelop/kdevelop/-/blob/master/kdevplatform/language/assistant/staticassistantsmanager.h?ref_type=heads#L53)